frameworks/cassandra/src/main/dist/cassandra_service.yml

```diff
@@ -0,0 +1,103 @@
+name: {{FRAMEWORK_NAME}}
```

Comment:
Can you rename this file to `svc.yml` please.  So far all services use this as the default name for their ServiceSpec YAML files.